embulk-standards/src/main/java/org/embulk/standards/JsonParserPlugin.java

```diff
@@ -228,6 +255,18 @@ private PageBuilder newPageBuilder(Schema schema, PageOutput output) {
         return new PageBuilder(Exec.getBufferAllocator(), schema, output);
     }
 
+    private Value parseColumnValueWithOffsetInJsonPointer(String valueAsJsonString, String jsonPointer) {
+        try {
+            return jsonParser.parseWithOffsetInJsonPointer(valueAsJsonString, jsonPointer);
+        } catch (JsonParseException e) {
+            /*
+             * When JsonParseException is thrown, it would be an error that the given JSON pointer doesn't match with the JSON object.
+             * We would return NULL when the pointer doesn't match, not throw Exception.
+             */
```

Comment:
We may want to discuss about this -- what we should do in case of parsing errors. For example in case `stop_on_invalid_record: true` ?

Reply:
If we discuss the specification of this, we would need to consider about the case of "when the specified key (or JSON pointer) doesn't exist in a record with custom schema". Current version always returns NULL in both case inexistent key and pointer.

Reply:
As a result of F2F communication, in the first version, we concluded that the JSON parser returns NULL if the given pointer doesn't match within a JSON record, which means it's not needed to change current version. cc: @dmikurube

Reply:
Sounds. good. Just nice to leave more comments here -- from the viewpoint of application-level, not only from the Embulk-core's code-level.